client/src/Components/Footer.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+
+const Footer = () =>
+{
+    let currentYear = new Date().getFullYear();
+    const {t} = useTranslation();
+    const {ul1,ul2,ul3,ul4} = t('ulTranslation')
+
+    return(
+        <div className="bg-green-200 py-5 flex space-x-[1050px]">
+            <ul className="ml-10 space-y-4">
+                <Link to={'/'}>
+                    <li className="hover:underline transition cursor-pointer">{ul1}</li>
+                </Link>
+                <li className="hover:underline transition cursor-pointer">{ul2}</li>
+                <li className="hover:underline transition cursor-pointer">{ul3}</li>
+                <li className="hover:underline transition cursor-pointer">{ul4}</li>
```

Comment:
There is no difference in handling of `ul2, ul3, ul4`

Reply:
`<li className="hover:underline transition cursor-pointer">{ul2}</li>`
`<li className="hover:underline transition cursor-pointer">{ul3}</li>`
`<li className="hover:underline transition cursor-pointer">{ul4}</li>`

The only thing which is being handles here is the `lang` which the user has selected as per his choice, nothing else.

Reply:
`uli2` e.g. is a link still on homepage like `AboutUs`. Please populate the links with each `ul` value. Content can be empty but at least links should be established